packages/terra-graphs-docs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cerner/terra-graphs-docs",
+  "private": true,
+  "version": "0.0.0",
+  "description": "A react based dev-site that has documentation and example graphs",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-graphs.git",
+    "directory": "packages/terra-graphs-docs"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-graphs",
+    "carbon",
+    "graphs",
+    "UI",
+    "d3",
+    "d3-module",
+    "graphing"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-graphs/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-graphs#readme",
+  "dependencies": {
+    "@cerner/carbon-graphs":"^2.16.0"
```

Comment:
I think you might need to add a dependency on terra-graphs too.

Reply:
Should I put 0.0.0 for the version?

Reply:
Yeah, until it gets released.